tests/scripts/all.sh

```diff
@@ -692,179 +687,173 @@ make CC=gcc CFLAGS='-Werror -Wall -Wextra'
 msg "test: AES_FEWER_TABLES + AES_ROM_TABLES"
 make test
 
-if uname -a | grep -F Linux >/dev/null; then
-    msg "build/test: make shared" # ~ 40s
+################################################################
+#### 3. Various targets (adapted config)
+################################################################
+
+# 3.1 ARM targets (build only)
+#################
+
+msg "build: arm-none-eabi-gcc, make" # < 10s
+cleanup
+cp "$CONFIG_H" "$CONFIG_BAK"
+scripts/config.pl baremetal
+make CC=arm-none-eabi-gcc AR=arm-none-eabi-ar LD=arm-none-eabi-ld CFLAGS='-Werror -O1' lib
+
+msg "build: arm-none-eabi-gcc -DMBEDTLS_NO_UDBL_DIVISION, make" # < 10s
+cleanup
+cp "$CONFIG_H" "$CONFIG_BAK"
+scripts/config.pl baremetal
+scripts/config.pl set MBEDTLS_NO_UDBL_DIVISION
+make CC=arm-none-eabi-gcc AR=arm-none-eabi-ar LD=arm-none-eabi-ld CFLAGS='-Werror -O1' lib
+echo "Checking that software 64-bit division is not required"
+! grep __aeabi_uldiv library/*.o
```

Comment:
Preexisting: if `! grep __aeabi_uldiv library/*.o` fails, it will abort even in keep-going mode. The next line should be `if_build_succeeded ! grep -l __aeabi_uldiv library/*.o`.

Reply:
Thanks, will fix.